topics/RDMA.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
+## Getting Started
+
+[RDMA](https://en.wikipedia.org/wiki/Remote_direct_memory_access)
+enables direct data exchange between networked computers' main memory,
+bypassing processors and operating systems.
+
+As a result, RDMA offers better performance compared to TCP/IP. Test results indicate that
+Valkey Over RDMA achieves approximately 2 times higher QPS and lower latency.
+
+Please note that Valkey Over RDMA is currently supported only on Linux.
+
+## Running manually
+
+To run a Valkey server with RDMA mode:
+
+    ~# ./src/valkey-server --protected-mode no \
```

Comment:
I presume `~# ` is the value of PS1 for your shell.  Perhaps it should be removed here and below.

Reply:
> I presume `~# ` is the value of PS1 for your shell. Perhaps it should be removed here and below.

I notice `./valkey-server --port 6380 --replicaof 127.0.0.1 6379` in `topics/valkey.conf.md`, so you are right. thanks!